canonical-web-and-design/ubuntu.com
# Technical intro whitepaper and engage page

## Done

Added Ubuntu Pro intro engage page and takeover

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Reload the page until you see the "A technical intro to Ubuntu Pro" takeover
- Check that it matches [the brief](https://docs.google.com/spreadsheets/d/1ml7W0nLVTQs4rTMZcz4vDqi7hXTwFwsH7McjDtGxSAI/edit#gid=2113339190)
- Click through to the engage page
- Check that it matches [the brief](https://docs.google.com/spreadsheets/d/1ml7W0nLVTQs4rTMZcz4vDqi7hXTwFwsH7McjDtGxSAI/edit#gid=1271849439) and [the copy doc](https://docs.google.com/document/d/12yfS4IK3upN8_CCKbrEP_tFQ6boWY089L5DNFvClWpY/edit)

## Issue / Card

Fixes #7705
